src/backend/looped/system/self_globals.cpp

```diff
@@ -20,6 +20,17 @@ namespace big
 
 		self::rot = ENTITY::GET_ENTITY_ROTATION(self::ped, 2);
 
+		int original_heap_size = *g_pointers->m_gta.m_heap_size / 1024 / 1024; //Compiler will optimize this out to right shift by 20 instead of 2 divides.
+		if (original_heap_size != 650)
+		{
+			LOG(INFO) << "Original Memory Heap Size: " << original_heap_size << "MB";
+			DWORD old_heap_protect;
+			VirtualProtect(g_pointers->m_gta.m_heap_size, 4, PAGE_READWRITE, &old_heap_protect);
+			*g_pointers->m_gta.m_heap_size = 650 * 1024 * 1024;
+			VirtualProtect(g_pointers->m_gta.m_heap_size, 4, old_heap_protect, &old_heap_protect);
+			LOG(INFO) << "Modified Memory Heap Size To: 650MB";
+		}
+
```

Comment:
Isn't this only helpful when injecting VERY early.

Reply:
Should we remove it?

Reply:
Yep this piece of code only works when you inject quite early during the game initialization, can be removed for now and added in a proper place later